controllers/imagepolicy_controller.go

```diff
@@ -174,7 +175,7 @@ func (r *ImagePolicyReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	}
 	r.event(pol, events.EventSeverityInfo, msg)
 
-	return ctrl.Result{}, nil
+	return ctrl.Result{}, err
```

Comment:
Not sure of the intent of this change, though it doesn't change the meaning of the code as it stands.

Reply:
Likely a relic from previous changes made that are no longer part of this PR. Should probably be changed back to `nil`.